### PR TITLE
CSS cleanup

### PR DESCRIPTION
**Pull request by mrmrs**
Small fixes to the CSS.
- Makes formatting of the file consistent
- Removes unnecessary code
- Corrects some simple syntax errors

### CODE REVIEW

File: www/_static/swift.css [hunk 1]
```diff
@@ -1,64 +1,104 @@
 /* Common styles */
-.body { color: black; background: white; margin: 0 0 0 0 }
+.body {
+  color: black;
+  background: white;
+  margin: 0 0 0 0;
```
Comment (philipgiuliani) [inline www/_static/swift.css]: Maybe replace this with `margin: 0`

File: www/_static/swift.css [hunk 2]
```diff
@@ -1,64 +1,104 @@
...
+  clear: right;
+}
+
+table {
+  border: 2px solid black;
+  border-collapse: collapse;
+  margin-top: 1em;
+  margin-left: 1em;
+  margin-right: 1em;
+  margin-bottom: 1em;
```
Comment (philipgiuliani) [inline www/_static/swift.css]: Instead of setting every margin to 1em, why not just set `margin: 1em`?

File: www/_static/swift.css [hunk 3]
```diff
@@ -1,64 +1,104 @@
...
-  margin: 1em 0pt 0pt 15pt; 
+  margin: 1em 0 0 15pt;
 }
 
-
-pre.grammar, pre.stdlib {
+pre.grammar,
+pre.stdlib {
   background: #FFFFE0;
   padding: 1em 0 0 0;
```
Comment (philipgiuliani) [inline www/_static/swift.css]: Can be `padding: 1em 0 0`

File: www/_static/swift.css [hunk 4]
```diff
@@ -73,5 +113,5 @@ pre.example {
   width: 33%;
   background: #C0C0FF;
   padding: 1em 1em 1em 1em;
```
Comment (philipgiuliani) [inline www/_static/swift.css]: `padding: 1em`

File: www/_static/swift.css (hunk 2 above)
Comment (mrmrs) [inline www/_static/swift.css]: I prefer being explicit. I prefer this notation over short hand so didn't want to change it.

File: www/_static/swift.css (hunk 3 above)
Comment (mrmrs) [inline www/_static/swift.css]: Again I prefer being explicit, didn't seem worth changing.

File: www/_static/swift.css (hunk 2 above)
Comment (philipgiuliani) [inline www/_static/swift.css]: Ok, but sometimes you used `padding: 1em 1em 1em 1em ` and sometimes you split it up. I would like to see a consistency here!

Comment (mrmrs) [inline www/_static/swift.css]: Those aren't my declarations. These might be intentional choices by the author that I don't think are worth changing. The PR promotes a bunch consistency. If you want to see that change I suggest opening up a separate PR.

File: www/_static/swift.css (hunk 3 above)
Comment (coobersdoobers) [inline www/_static/swift.css]: Who cares? I prefer press a spacebar and put a value, instead of selecting one and changing it. @philipgiuliani +1

Comment (mrmrs) [inline www/_static/swift.css]: @coobersdoobers I care. That is a code editing problem, not a css architecture problem :wink: 
I prefer explicitness, but changing shorthand format vs longform format for padding/margin wasn't part of the changes I wanted to make in this pull request. If you'd like to see those changes in I'd suggest opening up a new PR.

**Comment by tcyrus** [comment]
:+1:

**Comment by chrislloyd** [comment]
:+1: on @mrmrs' changes. @philipgiuliani your comments warrant discussion but aren't useful in the context of these changes. Perhaps open another PR if you'd like to collapse CSS rules?

File: www/_static/swift.css (hunk 1 above)
Comment (johnantoni) [inline www/_static/swift.css]: +1

That'd be better.

File: www/_static/swift.css (hunk 4 above)
Comment (johnantoni) [inline www/_static/swift.css]: +1 cleaner

**Comment by CodaFi** [comment]
This issue is now resolved by the merge of #3368.  Thank you @mrmrs for your contribution, and thank you to everybody else here for the discussion.

**Pull request closed without merging**